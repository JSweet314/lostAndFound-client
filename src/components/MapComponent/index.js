import React from 'react';
import { withScriptjs, withGoogleMap, GoogleMap } from 'react-google-maps';

export const MapComponent = withScriptjs(withGoogleMap(() => {
  return (
    <GoogleMap
      defaultZoom={16}
      defaultCenter={{ lat: 39.7508, lng: -104.9966 }}>  
    </GoogleMap>
  );
}));

export default MapComponent;