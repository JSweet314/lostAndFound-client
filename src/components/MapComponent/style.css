.map-loading {
  width: 100%;
  height: 100%;
  background-image: url('../../images/Loadingsome.gif');
  background-size: 50%;
  background-repeat: no-repeat;
  background-position: center;
  position: absolute;
  top: 0;
}

.map-loading h3 {
  text-align: center;
  font-size: 3rem;
  margin: 5rem 0;
}