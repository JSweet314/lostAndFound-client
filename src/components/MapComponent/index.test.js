import React from 'react';
import { shallow } from 'enzyme';
import MapComponent from './index';
import { googleUrl } from '../../private/keys';

describe('MapComponent', () => {
  it('should match a snapshot', () => {
    const wrapper = shallow(
      <MapComponent
        loadingElement={<div></div>}
        googleMapURL={googleUrl}
      />
    );
    expect(wrapper).toMatchSnapshot();
  });
});