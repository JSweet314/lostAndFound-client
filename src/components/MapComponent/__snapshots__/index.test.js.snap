// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MapComponent should match a snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <withScriptjs(withGoogleMap(Component))
    googleMapURL="https://maps.googleapis.com/maps/api/js?key=AIzaSyCVepTj3yZKdBV-t1k2wz5aHkbKYWFPL0w&v=3.exp&libraries=geometry,drawing,places"
    loadingElement={<div />}
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {},
    "ref": null,
    "rendered": null,
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {},
      "ref": null,
      "rendered": null,
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
